doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -232,10 +232,21 @@ Or::
 The best way to deal with a full cluster is to add new ``ceph-osds``, allowing
 the cluster to redistribute data to the newly available storage.
 
-If you cannot start an OSD because it is full, you may delete some data by deleting
-some placement group directories in the full OSD.
+If you cannot add a new OSD, you can do the following:
```

Comment:
the pg won't be transferred back if the osd is above the backfill_full threshold..